code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -1811,3 +1811,208 @@
 				continue
 			L.adjustSanityLoss(-heal_amount)
 			new /obj/effect/temp_visual/healing(get_turf(L))
+
+/obj/item/ego_weapon/blind_obsession//When I saw that Ishmael's version was an anchor I thought "hey would it be funny if it was a throwing weapon with aoe".
+	name = "blind obsession"
+	desc = "All hands, full speed toward where the lights flicker. The waves... will lay waste to everything in our way."
+	special = "This weapon requires two hands to use. \
+			Use in hand to unlock its full power for a short period of time at the cost of speed. \
+			When at thrown at full power, this weapon damages everyone but yourself in an AOE. Be careful! \
+			This weapon deals 75% more damage on fully powered direct throws."
+	icon_state = "blind_obsession"
+	lefthand_file = 'icons/mob/inhands/64x64_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/64x64_righthand.dmi'
+	force = 80
+	attack_speed = 2.5
+	throwforce = 80
+	throw_speed = 1
+	throw_range = 9
+	damtype = RED_DAMAGE
+	attack_verb_continuous = list("bashes", "smashes")
+	attack_verb_simple = list("bashes", "smashes")
+	hitsound = 'sound/weapons/ego/hammer.ogg'
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 80
+							)
+	var/charged
+	var/speed_slowdown = 0
+	var/mob/current_holder
+
+
+//Equipped setup
+/obj/item/ego_weapon/blind_obsession/equipped(mob/living/carbon/human/user, slot)
+	. = ..()
+	if(!user)
+		return
+	current_holder = user
+	RegisterSignal(current_holder, COMSIG_MOVABLE_MOVED, .proc/UserMoved)
+
+//Destroy setup
+/obj/item/ego_weapon/blind_obsession/Destroy(mob/user)
+	if(!user)
+		return ..()
+	speed_slowdown = 0
+	UnregisterSignal(current_holder, COMSIG_MOVABLE_MOVED)
+	current_holder = null
+	return ..()
+
+//Dropped setup
+/obj/item/ego_weapon/blind_obsession/dropped(mob/user)
+	. = ..()
+	if(!user)
+		return
+	speed_slowdown = 0
+	UnregisterSignal(current_holder, COMSIG_MOVABLE_MOVED)
+	current_holder = null
+
+/obj/item/ego_weapon/blind_obsession/proc/UserMoved(mob/user)
+	SIGNAL_HANDLER
+	user.add_or_update_variable_movespeed_modifier(/datum/movespeed_modifier/anchor, multiplicative_slowdown = speed_slowdown)
+
+/obj/item/ego_weapon/blind_obsession/CanUseEgo(mob/living/user)
+	. = ..()
+	if(user.get_inactive_held_item())
+		to_chat(user, span_notice("You cannot use [src] with only one hand!"))
+		return FALSE
+
+/obj/item/ego_weapon/blind_obsession/attack_self(mob/user)
+	if(user.get_inactive_held_item())
+		to_chat(user, span_notice("You cannot impower [src] with only one hand!"))
+		return
+	if(do_after(user, 12, src))
+		charged = TRUE
+		speed_slowdown = 2
+		throwforce = 100//TIME TO DIE!
+		to_chat(user,span_warning("You put your strength behind this attack."))
+		addtimer(CALLBACK(src, .proc/PowerReset), 18,user)//prevents storing 3 powered up anchors and unloading all of them at once
```

Comment:
What do you mean 'storing'? All of these values set, not add or multiply. So it doesn't matter if you can do it repeatedly. In fact you should just check if it's already charged and deny it, if that's the worry.